app/views/batch_record_updates/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Batch Record Updates</h1>
+<div class="home-page-section">
+  <h1>Batch Record Updates</h1>
+
+  <% if current_user %>
+  <h3>What would you like to do?</h3>
+  <div id="batch-record-menu">
+    <ul class="sub-title">
+      <li><%= link_to "Upload a batch", '#', data: { toggle: 'show', show: '#upload-batch-menu', hide: '#show-queue-menu #show-errors' } %></li>
+      <%= render partial: 'upload_batch_menu' %>
+      <li><%= link_to "Show queue", '#', data: { toggle: 'show', show: '#show-queue-menu', hide: '#upload-batch-menu #show-errors' } %></li>
+      <%= render partial: 'show_queue_menu' %>
+      <li><%= link_to "Show errors", '#', data: { toggle: 'show', show: '#show-errors', hide: '#show-queue-menu #upload-batch-menu' }%></li>
+      <%= render partial: 'show_errors' %>
+    </ul>
```

Comment:
This seems like something that Bootstrap handles pretty well out-of-the-box (which could help in reducing the maintenance burden).

Check out [Bootstrap Accordion](http://getbootstrap.com/javascript/#collapse-example-accordion) or even [Bootstrap Tabs](http://getbootstrap.com/javascript/#tabs) for possible implementation ideas.